docs/UserGuide.adoc

```diff
@@ -31,95 +43,127 @@ Jump to <<Quick Start>> to get started. Enjoy!
 .  Copy the file to the folder you want to use as the home folder for your Cooking Papa.
 .  Double-click the file to start the app. The GUI should appear in a few seconds.
 +
+.CookingPapa User Interface
 image::Ui.png[width="790"]
+[caption="Figure 1: ]
 +
-.  Type the command in the command box and press kbd:[Enter] to execute it.
-e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
-.  Some example commands you can try:
-
-* *`example`* : include examples
 
 . Refer to <<Features>> for details of each command.
 
 [[Features]]
 == Features
 
-====
-*Command Format*
+CookingPapa accepts the following command formats for its application. It is important to follow these guidelines
+for smooth usage of the application.
 
-* Words in `UPPER_CASE` are the parameters to be supplied by the user e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
-* Items in square brackets are optional e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
-* Items with `…`​ after them can be used multiple times including zero times e.g. `[t/TAG]...` can be used as `{nbsp}` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
-* Parameters can be in any order e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
-====
+*Command Format*
 
+* Words in `UPPER_CASE` are the parameters to be supplied by the user. +
+(e.g. in ``add   i/INGREDIENT``, ``INGREDIENT``
+is a parameter which can be used as ``add i/Bacon``).
+* Items in square brackets are optional. +
+ (e.g `recipe add n/NAME [t/TAG]` can be used as `recipe add n/Cheesecake t/Dessert`
+or as `recipe add n/Cheesecake`).
+* Items with `…`​ after them can be used multiple times. +
+(e.g. `add [i/INGREDIENT] [q/QUANTITY]...`
+can be used to add multiple ingredients and its' respective quantities in the same command).
+* Parameters can be in any order. +
+(e.g. if the command specifies `n/NAME t/TAG i/INGREDIENT q/QUANTITY`,
+`q/QUANTITY t/TAG i/INGREDIENT` is also acceptable).
+
+[[Cookbook]]
 === Cookbook Commands
 Cookbook commands are commands that allow users to add and remove recipes from the cookbook,
-as well as view recipes stored in the cookbook. More commands that can be done with recipes
+as well as view recipes stored in the cookbook. More commands can be done with recipes and
 are covered in <<Recipe commands>>.
 
 ==== View a recipe from the cookbook
 This command allows you to view a recipe, the ingredients required, and the steps to cook it.
 
 - Format: `cookbook view INDEX`
 - Example:
-* `cookbook view 1` +
-    Views the details of recipe 1.
+|===
+| Parameters | Result
```

Comment:
Maybe we can change "Parameters" to "Examples" instead?

Reply:
The entire table is shown under example, so parameters are the examples of parameters. Hope that makes sense!

Reply:
or would "Command" be better? cus parameters seems like it's only referring to the arguments